modules/iframe-app/config-node.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This should probably be `.gitignored` like our other config files

Reply:
Definitely! Fixed.